### PR TITLE
Enable the messaging framework in nightly

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: nimbus.fml.yaml [hunk 1]
```diff
@@ -123,6 +123,8 @@ features:
             DEVICE_IOS:               os == 'iOS'
             ALWAYS:                   "true"
             NEVER:                    "false"
+            I_AM_DEFAULT_BROWSER: "is_default_browser_string == 'true'"
+            I_AM_NOT_DEFAULT_BROWSER: "is_default_browser_string == 'false'"
```
Comment (jhugman) [inline nimbus.fml.yaml:127]: Nit: whitespace

```suggestion
            I_AM_DEFAULT_BROWSER:          is_default_browser_string == 'true'
            I_AM_NOT_DEFAULT_BROWSER: is_default_browser_string == 'false'
```

File: nimbus.fml.yaml [hunk 2]
```diff
@@ -165,61 +167,18 @@ features:
...
-      - channel: developer
+      - channel: nightly
         value: {
           "messages": {
-            "my-viewpoint-survey": {
-              "title": "Message tile",
-              "text": "Love Firefox? Fill in our survey!",
-              "action": "https://surveyprovider.com/survey-id/{uuid}",
-              "trigger": [ "ALWAYS" ],
+            "default-browser-en": {
```
Comment (jhugman) [inline nimbus.fml.yaml:173]: Could you convert this to YAML?

File: nimbus.fml.yaml [hunk 3]
```diff
@@ -165,61 +167,18 @@ features:
...
+              "text": "Set links from websites, emails, and messages to open automatically in Firefox.",
```
Comment (jhugman) [inline nimbus.fml.yaml:174]: We can use the existing strings:

```yaml
messages:
  default-browser:
    text:         default_browser_experiment_card_text
    button-label: preferences_set_as_default_browser 
```

and target everyone, not just the EN speakers.

File: app/src/main/java/org/mozilla/fenix/FeatureFlags.kt [hunk 4]
```diff
@@ -113,6 +113,5 @@ object FeatureFlags {
     /**
      * Enables receiving from the messaging framework.
      */
-    @Suppress("MayBeConst")
-    val messagingFeature = false
+    val messagingFeature = Config.channel.isNightlyOrDebug
```
Comment (jhugman) [inline app/src/main/java/org/mozilla/fenix/FeatureFlags.kt:116]: 😻

**Review by jhugman — APPROVED** [review]
This is beginning to look like a real messaging system! Well done and thank you.

File: nimbus.fml.yaml (hunk 3 above)
Comment (Amejia481) [reply nimbus.fml.yaml:174]: 🙌🏽

File: nimbus.fml.yaml (hunk 2 above)
Comment (Amejia481) [reply nimbus.fml.yaml]: Done!!

**Pull request merged**